### ISSUE TITLE
Postfix module does not respect Horizontal layout

### ISSUE THREAD

**Issue by shoulders** [webmin/authentic-theme#1702]
| SYSTEM INFORMATION ||
|----------|----------|
|  OS type and version |  Ubuntu Linux 22.04.4             |
|  Usermin version     |  2.102                            |
|  Virtualmin version  |  7.20.2                           |
|  Theme version       |  21.20.7                          |
|  Apache version      |  2.4.52                           |
|  Package updates     |  9 package updates are available  |

### The issue

The horizontal layout still has 2 columns in places and makes it not easy to use. here is an example from the general options page.

![image](https://github.com/user-attachments/assets/e856f6b7-e27b-4c3b-a3dc-28402adbeca0)

**Postfix module pages that seem affected**
- General Options
- Address Rewriting and Masquerading
- General Resource Control
- SMTP Server Options
- SMTP Client Options
- Delivery rates

### Additional

- I hope authentic theme was the right place for this issue. 😄 
- I have not checked any other modules for this issue.
- I can check all of the other modules I have access or open a new ticket as I find them if you want.
- I know that changing the theme to a `horizontal style` has been pencilled in for Webmin 2.3 so I was unsure if I should report this bug/issue

**Comment by iliajie** [comment]
Hello,

Yes, this is most likely because Postfix doesn’t use the Webmin UI library to render HTML.

**Comment by shoulders** [comment]
aah, more than a few lines of code to fix then.

the postfix module has many bugs, it needs a bit of TLC, 😄

**Comment by shoulders** [comment]
bump - Can I get this reopened and re-assessed.

**Comment by jcameron** [comment]
FYI, the Postfix module uses the same UI code as the rest of Webmin.

**Comment by iliaross** [comment]
Agreed.

@shoulders — do you still have this issue?

**Comment by shoulders** [comment]
I will double check this and my other authtic theme issue, later when the sun goes down.

**Comment by shoulders** [comment]
not forgotten, we have nice weather in UK for once so I am digging a wall. I will be in the office tomorrow and will chase this up.

The wall beat me.

**Comment by iliaross** [comment]
I don't think this is an issue anymore with the latest Webmin 2.641?

**Comment by shoulders** [comment]
I will have a look tomorrow, sorry for the delay I discovered vibe coding.

**Comment by shoulders** [comment]
stupid question, how do i access the widget? there does not appear to be a horizontal/vertical selection in theme settings.  So some clarification so I can properly evaluate this issue.
- The widget has been removed? I cant find the toggle to unhide it either.
    -  it use to be here `Theme Configuration --> Configuration category: Table display --> Show content layout control: Yes`
- There is only one layout option, which replaces horizontal/vertical: Force single column mode
    - Theme Configuration --> Configuration category: Content page --> Force single column  mode

And then I will check over the pages I flagged.

**Comment by iliaross** [comment]
> stupid question, how do i access the widget? there does not appear to be a horizontal/vertical selection in theme settings. So some clarification so I can properly evaluate this issue.

The widget is no longer available. You should evaluate it based on its appearance in the latest Webmin version.

**Comment by shoulders** [comment]
> The widget is no longer available. You should evaluate it based on its appearance in the latest Webmin version.

This answers question 1

There is only one layout option, which replaces horizontal/vertical: Force single column mode

Can you clarify this one as I think this is extensively what horizontal/vertical did. 

thanks

**Comment by iliaross** [comment]
I'm confused and not sure what you’d like me to clarify?

**Comment by shoulders** [comment]
is the `horizontal/vertical toggle` that was in the widget equivalent to the `Force single column mode` option, i think it probably is.

**Comment by iliaross** [comment]
No, it isn’t. It’s still a new column mode—one is the default with two columns, and the other is optional with one.

**Comment by shoulders** [comment]
Ok, but that's exactly how the horizontal/vertical work. One had 2 columns and then the other was single column

I will experiment and get this looked at asap.

Thanks

**Comment by shoulders** [comment]
So I have gone through all of the pages in the standard 2 column mode (i.e. not single column mode) and everything was in nice neat columns. no bleeding. so this issue has been fixed.

Loosing the widget was a good thing and putting the setting directly in the theme settings is a good thing.
